scenario.scala

```diff
@@ -38,14 +38,22 @@ object Walk {
     .get("/electronics/")
     .headers(headers_classic)
 
-  val categoryListFilterPrice = http("Category list filtering")
+  val categoryListFilterPrice = http("Category list filtering by price")
     .get("/electronics/?product_filter_form[minimalPrice]=0&product_filter_form[maximalPrice]=20000")
     .headers(headers_ajax)
 
-  val categoryListFilterPriceBrand = http("Category list filtering")
+  val categoryListFilterPriceBrand = http("Category list filtering by price with brand selected")
     .get("/electronics/?product_filter_form[minimalPrice]=0&product_filter_form[maximalPrice]=20000&product_filter_form[brands][]=4")
     .headers(headers_ajax)
 
+  val categoryListFilterParameters = http("Category list filtering by parameters")
+    .get("/electronics/?=product_filter_form[parameters][3][]=6&product_filter_form[parameters][4][]=8&product_filter_form[parameters][4][]=12")
```

Comment:
```suggestion
    .get("/electronics/?product_filter_form[parameters][3][]=6&product_filter_form[parameters][4][]=8")
```

Reply:
also looks like there is no USB parameter option 12 and there seems to be a BUG when one parameter doesn't exist, so the existing one is not checked

Reply:
maybe pick some more complex combination